bsip-0058.md

```diff
@@ -67,17 +67,221 @@ related poll worker proposals, including FOR and AGAINST, will be created for vo
+
+
+##### <div id="gsp-activated-trading" />Trading
+
+A smart asset can be traded against its collateral as long as the asset issuer has not restricted the markets against which it can be traded.  In the case of the smart assets that are issued by the BitShares Committee (e.g. bitCNY, bitUSD, etc.), these assets will continue to be available for trading on the DEX.
+
+Margin calls will still be present on the DEX order book during GSP.  _This allows holders of the smart asset to trade for often more collateral than is possible to obtain than by force-settling their holdings._
+
+Furthermore, the GSP price is dynamically being updated as (a) margin calls are filled, and (b) collateral behind debt positions are updated by borrowers.  **These updates will permit the dynamic GSP price to move towards the natural feed price.**  The consequence is that margin calls might be present on the order books and provide additional offers for holders to consider.
+
+|Comparison to Globally Settled Asset|
+|-|
+|Trading of a globally settled asset is also possible.  _Under GSP, the possible presence of margin calls on the DEX order books provides holders with more collateral per smart asset (e.g. more BTS per bitCNY) than under under global settlement. Futhermore, the dynamic GSP adjusts the price of these margin calls which provides holders with collateral offers/prices that are higher than with the force settlement price under global settlement._|
+
+
+##### <div id="gsp-activated-force-settling" /> Force-Settling
+
+Besides using the DEX to exchange a smart asset (e.g. bitCNY) to trade for its backing collateral (e.g. BTS), a holder of a smart asset has the ability "force-settle" (FS) a smart asset directly into the collateral subject to a delay of 24 hours and subject to a penalty that is specific to a smart asset (e.g. 5%).  This FS ability is true both when an asset is globally settled or not.  When it is not globally settled, the force settlement price of D units of a smart asset can be force settled for collateral equal to
+
+K = D &times; FSR &div; FP<sub>P</sub><sup>K/D</sup> = D &times; FSR &times; FP<sub>P</sub><sup>D/K</sup>
+
+where
+
+- D: amount of the smart asset that is held
+- FSR: force-settlement ratio (e.g. 95%)
+- FP<sub>P</sub><sup>K/D</sup>: published feed price 24 hours after the force settlement is initiated denominated in terms of smart asset divided by collateral (e.g. 2 bitCNY per BTS)
+- FP<sub>P</sub><sup>D/K</sup>: published feed price 24 hours after the force settlement is initiated denominated in terms of collateral divided by smart asset (e.g. 0.5 BTS per bitCNY)
+
+When a smart asset is globally settled, it is still possible to force settle with two differences in comparison to normal operations:
+
```

Comment:
There is another important difference: after GS, the settlement offset is ignored, i. e. settlement happens at the settlement price (not 5% below it).